.gitignore

```diff
@@ -16,3 +16,5 @@ target/
 .vscode/settings.json
 
 *.iml
+
+showcase/gapic-showcase/src/test/resources/compliance_suite.json
```

Comment:
I think we should check it in if it is required for testing.

Reply:
Ok, I can add it in.